IPython/core/application.py

```diff
@@ -43,6 +43,13 @@
         "/etc/ipython",
     ]
 
+_envvar = os.environ.get('IPYTHON_SUPPRESS_ERRORS','').lower()
+if _envvar == 'true':
```

Comment:
The two conventions I'm most used to for boolean environment variables are either 1/0, or anything but the empty string means true.